thunder/__init__.py

```diff
@@ -637,16 +638,7 @@ def get_computation_and_inputs(*args, **kwargs):
 
             if backward_trc is None:
                 ## EPILOGUE and TRANSFORMS should not mix...
```

Comment:
can you remove this comment, too please?
also the cs.last_computation_transformation_start/stop could be removed both from the computation here and indeed from the compile stats class.
Finally, we want to drop the `transform_additionally` from the `Transform` class, as it will not be used after this PR.

Reply:
yes absolutely! (wanted to remove it after I was sure that `transform_additionally` can be completely removed :)